Components/CategoryResolver/AutoCategoryResolver.php

```diff
@@ -124,6 +124,28 @@ public function convertTreeToEntities(array $node, Category $parent = null, $cat
     }
 
     /**
+     * Loop through category tree and fetch ids
+     *
+     * @param array $node
+     * @param array $categories
+     * @return array
+     */
+    public function convertTreeToKeys(array $node, $categories = [])
```

Comment:
This could be private function in ImportService, otherwise we have to add it in CategoryResolver interface and implement it in both CategoryResolvers.